edx_proctoring/static/index.js

```diff
@@ -0,0 +1,32 @@
+export const handlerWrapper = (Handler) => {
+   let handler = new Handler({});
+
+  self.addEventListener("message", (message) => {
+    switch(message.data.type) {
+      case 'config': {
```

Comment:
Documentation on this event in the backends.rst file would be helpful, even if its just to state we will call your initializer with the options passed into this event.

Reply:
You're completely correct that this is weird; I only have this in here because of a dev-only need. We won't be able to set up a sandbox server for our testing-only reference implementation without this in here. I didn't document it because I don't actually want vendor implementations to depend on it.

Reply:
Makes sense - but maybe add a comment anyways so others (internally) don't accidentally build on it